xep-0280.xml

```diff
@@ -341,6 +341,7 @@
   <body>Neither, fair saint, if either thee dislike.</body>
   <thread>0e3141cd80894871a68e6fe6b1ec56fa</thread>
   <private xmlns='urn:xmpp:carbons:2'/>
+  <no-copy xmlns="urn:xmpp:hints"/>
```

Comment:
Not that it really matters, but we should probably use single vs. double quotes consistently.